catboost/libs/fstr/calc_fstr.h

```diff
@@ -134,14 +134,16 @@ TVector<TVector<double>> GetFeatureImportances(
     const TFullModel& model,
     const NCB::TDataProviderPtr dataset, // can be nullptr
     int threadCount,
-    int logPeriod = 0);
+    int logPeriod = 0,
+    TString shapMode = "Auto");
```

Comment:
Let's remove this default and move this parameter after `threadCount` here and below.